.github/ISSUE_TEMPLATE/feature-request-error-stack.yml

```
@@ -0,0 +1,49 @@
+name: Feature request (error-stack)
```

```suggestion
name: Suggestion or request (error-stack)
```

I think the description and form itself imply more than just 'feature request', so something broader like this may be a more appropriate title.

---

Good point, applied that in https://github.com/hashintel/hash/pull/1145/commits/45a8379ca621133ca9a22a3d3e1225d2086645e4